.travis.yml

```diff
@@ -12,7 +12,6 @@ php:
 
 matrix:
   allow_failures:
-    - php: 7.1
```

Comment:
Sorry, I missed that because I branched from a previous commit. I was going to fix that, but a single row addition will not be enough: we have 8 total 7.x builds, since we have the `DISABLE_ASSERTIONS` ones... are those still needed? Can we remove them?